ui/component/or-rules/src/json-viewer/or-rule-json-viewer.ts

```diff
@@ -120,8 +120,8 @@ export class OrRuleJsonViewer extends translate(i18next)(LitElement) implements
     @property({attribute: false})
     protected _ruleset!: RulesetUnion;
 
-    @state()
-    protected _loadedAssets: Map<string, Asset[]> = new Map<string, []>();
+    @state() // to be exact: Map<Realm name, Map<AssetType name, Asset[]>>
+    protected _loadedAssets: Map<string, Map<string, Asset[]>> = new Map<string, Map<string, Asset[]>>();
```

Comment:
Components shouldn't be aware of different realms; this is a higher level function that is only applicable to super users; we shouldn't optimise for this scenario as most users will be regular users working within a single realm, don't mind if there is a performance hit switching realms.

I have always tried to avoid complexity of making components "realm aware". When the realm changes then there should be an ability to clear loaded assets so a `reset()/refresh()` function of some kind